components/menu/index.css

```diff
@@ -10,170 +10,637 @@ OF ANY KIND, either express or implied. See the License for the specific languag
   }
+
+  /* remove inline-start spacing once an item is selected and a checkmark appears */
+  .spectrum-Menu-item--is-selected {
+    --spectrum-menu-checkmark-display: var(--spectrum-menu-checkmark-display-shown);
+    padding-inline-start: var(--mod-menu-item-label-inline-edge-to-content, var(--spectrum-menu-item-label-inline-edge-to-content));
+}
+}
+
+/* menus with "selectable" menuitems */
+/* provide inline-start spacing for potential checkmarks */
+/* .spectrum-Menu-item--is-selectable {
+  --spectrum-menu-checkmark-display: var(--spectrum-menu-checkmark-display-hidden);
+
+  padding-inline-start: var(--mod-menu-item-selectable-edge-to-text-not-selected, var(--spectrum-menu-item-selectable-edge-to-text-not-selected));  
+} */
+
+/* remove inline-start spacing once an item is selected and a checkmark appears */
+/* .spectrum-Menu-item--is-selected {
+  --spectrum-menu-checkmark-display: var(--spectrum-menu-checkmark-display-shown);
+
+  padding-inline-start: var(--mod-menu-item-label-inline-edge-to-content, var(--spectrum-menu-item-label-inline-edge-to-content));
+} */
+
+.spectrum-Menu-itemIcon {
+  fill: var(--highcontrast-menu-item-label-icon-color-default, var(--mod-menu-item-label-icon-color-default, var(--spectrum-menu-item-label-icon-color-default)));
 }
 
 .spectrum-Menu-checkmark {
-  transform: scale(1);
+  align-self: center;
+  display: var(--mod-menu-checkmark-display, var(--spectrum-menu-checkmark-display));
+  fill: var(--highcontrast-menu-checkmark-icon-color-default, var(--mod-menu-checkmark-icon-color-default, var(--spectrum-menu-checkmark-icon-color-default)));
   opacity: 1;
 }
 
-.spectrum-Menu-item {
+.spectrum-Menu-item,
+.spectrum-Menu li:not(.spectrum-Menu-item, .spectrum-Menu-divider) {
   cursor: pointer;
   position: relative;
-  display: flex;
   align-items: center;
 
   box-sizing: border-box;
 
-  padding: var(--spectrum-listitem-texticon-padding-y)
-    var(--spectrum-listitem-texticon-padding-right)
-    var(--spectrum-listitem-texticon-padding-y)
-    var(--spectrum-listitem-texticon-padding-left);
+  background-color: var(--mod-menu-item-background-color-default, var(--spectrum-menu-item-background-color-default));
+  
+  line-height: var(--mod-menu-item-label-line-height, var(--spectrum-menu-item-label-line-height));
 
+  min-block-size: var(--mod-menu-item-min-height, var(--spectrum-menu-item-min-height));
+  padding-block-start: var(--mod-menu-item-top-edge-to-text, var(--spectrum-menu-item-top-edge-to-text));
+  padding-block-end: var(--mod-menu-item-bottom-edge-to-text, var(--spectrum-menu-item-bottom-edge-to-text));
+  padding-inline: var(--mod-menu-item-label-inline-edge-to-content, var(--spectrum-menu-item-label-inline-edge-to-content));
+  
   margin: 0;
+  text-decoration: none;
+}
 
-  border-inline-start: var(--spectrum-listitem-texticon-focus-indicator-size) solid
-    transparent;
+.spectrum-Menu-item,
+.spectrum-Menu li:not(.spectrum-Menu-item, .spectrum-Menu-divider) {
```

Comment:
same as previous comment. Should `:not(.spectrum-Menu-item)` have the same rules as `.spectrum-Menu-item`?